src/openrct2/network/NetworkBase.cpp

```diff
@@ -49,7 +49,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-constexpr uint8_t kNetworkStreamVersion = 0;
+constexpr uint8_t kNetworkStreamVersion = 1;
```

Comment:
Why the bump in version? I don't see anything in the changes that suggest it would do anything differently

Reply:
I confused the changed intents with changed game actions. Thanks for pointing it out, I’ll revert this.